### ISSUE TITLE
Streamline 1.0 fails on CS comprehension idioms

### ISSUE THREAD

**Issue by bjouhier** [Sage/streamlinejs#302]
See #298 

Repro:

```coffee
nodes = (@createTestNode suite, _ for i in [0...numNodes])
```

And here's the JS that compiles to:

```js
nodes = (function() {
  var j, ref, results1;
  results1 = [];
  for (i = j = 0, ref = numNodes; 0 <= ref ? j < ref : j > ref; i = 0 <= ref ? ++j : --j) {
    results1.push(this.createTestNode(suite, _));
  }
  return results1;
}).call(_this);
```

**Comment by bjouhier** [comment]
@aseemk 
Streamline does not handle the idiom with `_this`, just `this`. Should be easy to fix.

**Comment by aseemk** [comment]
Thanks @bjouhier!

**Comment by bjouhier** [comment]
@aseemk You're welcome :smile: 

I'm going to stop here for the day. I'll fix #301 and #297 tomorrow and I will republish to NPM. I'll look at the other ones a bit later.
